cesium_app/tests/frontend/test_predict_raw_data.py

```diff
@@ -0,0 +1,19 @@
+from cesium_app.tests.fixtures import (create_test_project, create_test_dataset,
+                                       create_test_featureset, create_test_model)
+import requests
+import json
+
+
+def test_predict_raw_data():
+    with create_test_project() as p, create_test_dataset(p) as ds,\
+         create_test_featureset(p) as fs, create_test_model(fs) as m:
+        ts_data = [[1, 2, 3, 4], [32.2, 53.3, 32.3, 32.52], [0.2, 0.3, 0.6, 0.3]]
+        impute_kwargs = {'strategy': 'constant', 'value': None}
+        query_string = ('http://localhost:5000/predict_raw_data?ts_data={}'
```

Comment:
Let's not hardcode things like ports---I use a different port to test on my machine.